src/main/webapp/WEB-INF/tags/instructor/course/studentsTable.tag

```diff
@@ -0,0 +1,87 @@
+<%@ tag description="instructorCourseDetails - Course Information Board" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ attribute name="courseDetails" type="teammates.common.datatransfer.CourseDetailsBundle" required="true" %>
+<%@ attribute name="studentsTable" type="teammates.ui.template.CourseDetailsStudentsTable" required="true" %>
+
+<table class="table table-bordered table-striped">
+    <c:set var="hasSection">
+        ${courseDetails.stats.sectionsTotal > 0}
+    </c:set>
+
+    <thead class="fill-primary">
+        <tr>
+            <c:if test="${hasSection}">
+                <th onclick="toggleSort(this, 1);" id="button_sortstudentsection" class="button-sort-none">
+                    Section<span class="icon-sort unsorted"></span>
+                </th>
+            </c:if>
+            <th onclick="toggleSort(this, ${hasSection ? 2 : 1});" id="button_sortstudentteam" class="button-sort-none">
+                Team<span class="icon-sort unsorted"></span>
+            </th>
+            <th onclick="toggleSort(this, ${hasSection ? 3 : 2});" id="button_sortstudentname" class="button-sort-none">
+                Student Name<span class="icon-sort unsorted"></span>
+            </th>
+            <th onclick="toggleSort(this, ${hasSection ? 4 : 3});" id="button_sortstudentstatus" class="button-sort-none">
+                Status<span class="icon-sort unsorted"></span>
+            </th>
+            <th class="align-center no-print">
+                Action(s)
+            </th>
+        </tr>
+    </thead>
+    
+    <c:forEach items="${studentsTable.rows}" var="row" varStatus="i">
+        <tr class="student_row" id="student${i.index}">
+            <c:if test="${hasSection}">
+                <td id="<%=Const.ParamsNames.SECTION_NAME%>">${row.student.section}</td>
+            </c:if>
+            <td id="<%=Const.ParamsNames.TEAM_NAME%>">${row.student.team}</td>
+            <td id="<%=Const.ParamsNames.STUDENT_NAME%>">${row.student.name}</td>
+            <td class="align-center">
+                <c:choose>
+                    <c:when test="${row.student.registered}">
+                        <%= Const.STUDENT_COURSE_STATUS_JOINED %>
+                    </c:when>
+                    <c:otherwise>
+                        <%= Const.STUDENT_COURSE_STATUS_YET_TO_JOIN %>
+                    </c:otherwise>
+                </c:choose>
```

Comment:
Wasn't there an equivalent to this if-else segment in the Java side?
Is it possible to make use of that?